cloud/tasks/storage/common.go

```diff
@@ -244,6 +244,28 @@ func taskStateTableDescription() persistence.CreateTableDescription {
 	)
 }
 
+func readyToRunTableDescription() persistence.CreateTableDescription {
```

Comment:
readyToExecuteTableDescription

we generalize "run" and "cancel" terms with "execute" (see https://github.com/ydb-platform/nbs/blob/main/cloud/tasks/storage/storage_ydb_impl.go#L214)